extensions/positron-python/python_files/posit/positron/data_explorer.py

```diff
@@ -1957,9 +1964,15 @@ def _date_median(x):
     import pandas as pd
 
     # the np.array calls are required to please pyright
-    median_date = np.median(np.array(pd.to_numeric(x)))
-    out = pd.to_datetime(np.array(median_date), utc=True)
-    return out.tz_convert(x[0].tz)
+    median_value = np.median(np.array(pd.to_numeric(x)))
+
+    # if any datetime64 or datetimetz type in pandas
+    if x.dtype == "timedelta64[ns]":
```

Comment:
Seems like we're not using this function yet since summary stats aren't implemented, but once we are -- might have to watch out for dtypes like `timedelta64[s]` or other resolutions here

Reply:
True -- I had gotten the statistics minimally working but then decided to disable them for now (along with the sparklines) until we could review that specifically to decide what we want to show.